tests/unit/lib/observers/ParameterObserverTest_unittest.cc

```diff
@@ -41,6 +41,10 @@ class MockEmitter : public SGObject
 		observe<int32_t>(1, "a", 1, p);
 		observe<int32_t>(1, "b", 1, p2);
 		observe<int32_t>(1, "None", 1, p3);
+
+		std::function<int32_t()> lambda_function = []() { return 10; };
```

Comment:
I think this method should have something in it that changes its state. So that you later on can assert that it was *not* called

Reply:
It makes sense. I will update the tests accordingly.